### PR TITLE
Prepared for testing with pest

### PR DESCRIPTION
**Pull request by osteboy**
Alright, this is a "big" one.

tl;dr:
* Took the database file from https://github.com/ObjectRetros/arcturus-ms-3-5-base-db, converted it to sqlite (description inside the README in database/schema)
* Installed pest for testing, and upgraded some packages to allow installing pest 2 instead of pest 1
* Fixed some migrations which messed up for testing.
* Converted the existing tests to pest
* Removed tests which was not relevant for atomcms (came with the fortify possibly)

Some more details about the changes done to the migrations:

Removed foreign keys on a table which is about to be created does not make sense. They are only relevant if renaming the table. So i moved them all inside the conditional which checks if there already is an existing table from a previous cms.

When creating a new user, there's a if condition checking if we have given a referral code. This check should be wrapped with a `isset`, so i did. Found this issue via the tests

I modified the language seeder to only update the field if country code is found, not both country code and language. This was an issue with sqlite, but i think that it's not necessary to check for uniqueness on both of them anyway. It's just a seeder, the table has a unique on both fields anyway, so that should fail if there are duplicates

I also modified the team seeder, it looked up the duplication field with the wrong field name it seems

I've made a testing-schema.sql in database/schema which will populate the testing database with a emulator db. In this case arcturus morningstar. This is required to do tests.

### CODE REVIEW

**Review by DennisObject — APPROVED** [review]
LGTM 👍🏻 

Thank you so much for the contribution
